modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -104,12 +104,10 @@ $ ls -Z /home/kni/rhcos_image_cache
 $ podman run -d --name rhcos_image_cache \ <1>
```

Comment:
I think you need to comment this callout so that the code block will be formatted correctly (note that in the preview, the last three lines of the code block are unformatted). More info here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks

```suggestion
$ podman run -d --name rhcos_image_cache \// <1>
```